src/app/_utilities/formatDateTime.ts

```diff
@@ -1,20 +1,20 @@
 export const formatDateTime = (timestamp: string): string => {
-  const now = new Date()
-  let date = now
-  if (timestamp) date = new Date(timestamp)
-  const months = date.getMonth()
-  const days = date.getDate()
-  // const hours = date.getHours();
-  // const minutes = date.getMinutes();
-  // const seconds = date.getSeconds();
+  const now = new Date();
+  let date = now;
+  if (timestamp) date = new Date(timestamp);
 
-  const MM = months + 1 < 10 ? `0${months + 1}` : months + 1
-  const DD = days < 10 ? `0${days}` : days
-  const YYYY = date.getFullYear()
-  // const AMPM = hours < 12 ? 'AM' : 'PM';
-  // const HH = hours > 12 ? hours - 12 : hours;
-  // const MinMin = (minutes < 10) ? `0${minutes}` : minutes;
-  // const SS = (seconds < 10) ? `0${seconds}` : seconds;
+  // Array of full month names
+  const monthNames = [
+    "January", "February", "March", "April", "May", "June",
+    "July", "August", "September", "October", "November", "December"
+  ];
 
-  return `${MM}/${DD}/${YYYY}`
-}
+  const months = date.getMonth(); // 0-based index
+  const days = date.getDate();
+  const YYYY = date.getFullYear();
+
+  const monthName = monthNames[months]; // Get the full name of the month
+  const DD = days < 10 ? `0${days}` : days; // Add leading zero to day if needed
```

Comment:
FYI, you can use padStart for this 😉   https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart

Reply:
great! thanks